src/backend/d3d12/DepthStencilStateD3D12.cpp

```diff
@@ -0,0 +1,101 @@
+// Copyright 2017 The NXT Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+#include "backend/d3d12/DepthStencilStateD3D12.h"
+
+#include "common/BitSetIterator.h"
+
+namespace backend {
+namespace d3d12 {
+
+	static D3D12_STENCIL_OP StencilOp(nxt::StencilOperation op) {
```

Comment:
Not sure what you mean here. I copied this style from InputStateD3D12.cpp.

By anonymous namespace do you mean not scoping inside a namespace at all?